docs/api/types.rst

```diff
@@ -97,20 +97,20 @@ EdgeDB ``array``  maps onto the JavaScript ``Array``.
     const edgedb = require("edgedb");
 
     async function main() {
-      const conn = await edgedb.connect({
+      const pool = await edgedb.createPool({
```

Comment:
I think we should continue to call this `connect()`.  Technically, this is a backwards-compatible change as all the methods behave the same. @1st1?

Reply:
+1

Reply:
https://github.com/edgedb/edgedb-js/pull/132#discussion_r686373221